project/src/main/webapp/directory.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <meta charset="utf-8">
+    <meta name="Voter Central Candidate Directory Page"
+          content="The directory page of Voter Central, a website
```

Comment:
You shouldn't have to describe what Voter Central is in every file. Just one file, perhaps the landing page, should be enough.

Reply:
I think this isn't actually shown on a page, but is more of an accessibility label?

Reply:
Didn't see that before. But looks like the content tag should be replaced with slot since the former is deprecated
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/content

Reply:
jk I think the meta tag is more of a search engine thing that indexers look at?

Reply:
> Didn't see that before. But looks like the content tag should be replaced with slot since the former is deprecated
> https://developer.mozilla.org/en-US/docs/Web/HTML/Element/content

I think that might be for the `<content>` tag, rather than the content attribute of the `<meta>` tag.

Reply:
I believe it's used to specify metadata about the page: https://www.w3schools.com/tags/tag_meta.asp. The content attribute for the meta element appears to be different from the content element.